app/assets/stylesheets/dashboard.scss

```diff
@@ -328,7 +328,7 @@
   }
   .no-articles {
     text-align: center;
-    background: white;
```

Comment:
You should leave this line as a fallback, not all browsers support CSS variables.